frontend/src/libs/components/card/styles.module.scss

```diff
@@ -28,10 +28,6 @@
   overflow: hidden;
 }
 
-.no-image {
-  padding-left: 20px;
```

Comment:
the .no-image class initially caused profile settings sidebar icon to move away from the border: 
<img width="334" alt="Screenshot 2023-09-09 at 12 29 48" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/44b7c250-5d5b-46fd-ac57-f737e6a35284">

Reply:
That changes are from another PR, check if that functionality working properly before removing it

Reply:
@v0ldemar01 I made sure I have latest changes from development and after removing this class I immediately checked if it affected other instances of the card component - no effect so far.

Reply:
You were correct, it is needed but for a component that was not in the development when I was making changes, I'm going to return in in the next commit

Reply:
@v0ldemar01 added back the .no-image style. 
I found the [conversation](https://bsa2023.slack.com/archives/C05P563E2MN/p1694184984693109)
where there was an issue with removing this class. 
I added back to the button styling and tested by removing icons and images from various buttons in two different sidebars. Hope will be fine.

https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/d8569e9f-18ee-4d33-bd50-29871fa7dc44